src/frontend/apps/desk/src/features/header/Header.tsx

```diff
@@ -56,7 +56,7 @@ export const Header = () => {
           </LogoGouv>
           <StyledLink href="/">
             <Box $align="center" $gap="1rem" $direction="row">
-              <Image priority src={IconApplication} alt={t('Régie Logo')} />
+              <Image priority src={IconApplication} alt="" />
```

Comment:
```suggestion
              <Image priority src={IconApplication} />
```

Reply:
Thank you @AntoLC. It seems that using an `alt=""` is mandatory.
source: https://www.w3.org/WAI/tutorials/images/decorative/

Also, the Image component from 'next/image' requires it.